fb_campaign_metrics_roas.dashboard.lookml

```diff
@@ -0,0 +1,963 @@
+- dashboard: fb_campaign_metrics__roas
```

Comment:
Also it should be extending the base dashboard and inheriting filters from that :)